src/screens/Welcome/__tests__/__snapshots__/WelcomeCarousel.test.js.snap

```diff
@@ -53,6 +53,7 @@ exports[`WelcomeCarousel does not render pagination in small screen 1`] = `
     loopClonesPerSide={3}
     onSnapToItem={[Function]}
     renderItem={[Function]}
+    scrollEnabled={true}
```

Comment:
No code changes here. Must be a newer version of the package that powers the carousel is rendering slightly different info